log_outgoing_requests/models.py

```diff
@@ -60,12 +63,24 @@ class OutgoingRequestsLog(models.Model):
         null=True,
         help_text=_("The request headers."),
     )
+    req_body = models.TextField(
+        verbose_name=_("Request body"),
+        blank=True,
+        null=True,
```

Comment:
you never want to make text-based fields nullable, as checking for empty value is ambiguous - `None` or `""`. Stick to a single type here and remove the `null=True`.